apps/nextcloud/templates/services/nginx/route.yml.j2

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Route
+metadata:
+  namespace: "{{ project_name }}"
+  name: "nextcloud-nginx-{{ prefix }}"
+  labels:
+    env_type: "{{ env_type }}"
+    customer: "{{ customer }}"
+    app: "nextcloud"
+    service: "nginx"
+    version: "{{ nextcloud_nginx_image_tag }}"
+    route_prefix: "{{ prefix }}"
+    route_target_service: "app"
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    haproxy.router.openshift.io/disable_cookies: "true"
```

Comment:
Can you add a comment to explain why this annotation is required?

Reply:
I have no idea, you did this work 😂